types/formidable/Formidable.d.ts

```diff
@@ -12,17 +18,17 @@ declare class IncomingForm {
      */
     parse(request: IncomingMessage, callback: (err: any, fields: Fields, files: Files) => void): void;
 
-    once(name: "end", callback: () => void): void;
-    once(name: "error", callback: (err: any) => void): void;
+    once(eventName: 'end', listener: () => void): this;
+    once(eventName: 'error', listener: (err: any) => void): this;
 
-    on(name: "data", callback: (data: EventData) => void): void;
-    on(name: "error", callback: (err: any) => void): void;
-    on(name: "field", callback: (name: string, value: string) => void): void;
-    on(name: "fileBegin" | "file", callback: (formName: string, file: File) => void): void;
-    on(name: "progress", callback: (bytesReceived: number, bytesExpected: number) => void): void;
-    on(name: string, callback: () => void): void;
+    on(eventName: 'data', listener: (data: EventData) => void): this;
+    on(eventName: 'error', listener: (err: any) => void): this;
+    on(eventName: 'field', listener: (name: string, value: string) => void): this;
+    on(eventName: 'fileBegin' | 'file', listener: (formName: string, file: File) => void): this;
+    on(eventName: 'progress', listener: (bytesReceived: number, bytesExpected: number) => void): this;
+    on(eventName: string, listener: () => void): this;
 
-    emit(name: "data", data: EmitData): void;
+    emit(eventName: 'data', data: EmitData): boolean;
```

Comment:
the method returns `boolean` based on the extended class

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/events.d.ts#L555